cocos/core/root.jsb.ts

```diff
@@ -124,23 +124,22 @@ rootProto.createLight = function (LightCtor) {
 };
 
 rootProto.destroyLight = function (l) {
```

Comment:
Is recycleLight absent in native platform ?

Reply:
Native platform uses this interface, so it needs to be modified synchronously

Reply:
Also needs recycleLight implementation in `root.jsb.ts`